src/libDirectoryService/FinalBlockPreProcessing.cpp

```diff
@@ -1133,8 +1131,10 @@ bool DirectoryService::RunConsensusOnFinalBlockWhenDSBackup()
     // Create new consensus object
 
     // Dummy values for now
```

Comment:
comment need to be modified

Reply:
Looks like this wasn't addressed, but let's forget it for now